### PR TITLE
Apple home page

### PR DESCRIPTION
**Pull request by gango-anan**
## In this Project I :
- I created index.html file to house the web page structure and the main.css file to contain the general styling rules of the page
- Added empty html elements to the index.html file to form the structure of the page
- Styled the navigation bar to mimic the original page navigation bar
- Styled the main section area which contains the main content of the page
- Finally Styled the footer of the page
- Wrote the project README file to help guide the intended users of the project how to generally get started with the page

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @gango-anan  . Thanks for submitting you project!

Great thing this project has and has done well:
:100: Use of semantic HTML
:100: Great Acknowledgment! Love it! 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] LinkedIn link is broken in README

- [ ] There is no "aside" element in the website, the elements wrapped inside the tag would be better positioned inside the main section.

- [ ] The clone is not displaying the 4 image elements
image

- [ ] The image is not displaying correctly. These elements shouldn't appear. 
image

- [ ] Navbar hover effect has no rounded corners for that match the normal one.
image
image

### [OPTIONAL]  :yellow_circle:  

- [ ] I would suggest to leave out the XAMPP config and put it as a pre-requisite. This way you can focus on the steps that are required for the website to display locally. And mention only in the "how to run" section a step like: start server and run in localhost.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by gango-anan** [comment]
Thanks for the observations made, they were invaluable; all recommended changes have be done, waiting for the next verdict, thanks.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      <div class="promo-img">
+        <img src="images/featured-ad2.jpg" alt="iPhone 5c">
+      </div>
+      <div class="promo-img">
+        <img src="images/featured-ad3.jpg" alt="Happy bday, mac">
+      </div>
+      <div class="promo-img">
+        <img src="images/featured-ad4.jpg" alt="supplier responsibility">
+      </div>
+  </main>
```
Comment (SpaYco) [inline index.html]: - [ ] these should be links (to other articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] the first line of the footer, should have one section on the right and the other on the left, yours is all on the left

| Yours | Original |
| --- | --- |
| image | image |

- [ ] check more below 👇 



### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by gango-anan** [comment]
Fixed all the issues raised, awaiting approval, thanks.

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by gango-anan** [comment]
Project Approved, merging apple-home-page branch to the master branch

**Pull request merged**